src/domain/entities/measurement.py

```
@@ -0,0 +1,137 @@
+"""測定データのドメインエンティティ"""
+from datetime import datetime, timezone
+from enum import Enum
+from typing import Any, Dict, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)